CONTRIBUTING.md

```diff
@@ -26,45 +26,16 @@ series [How to Contribute to an Open Source Project on GitHub][egghead]
 > can make all of your pull request branches based on this `master` branch.
 > Whenever you want to update your version of `master`, do a regular `git pull`.
 
-## Add yourself as a contributor
```

Comment:
I know that cleaning up random things in an unrelated PR is poor practice, but I couldn't help myself 😅

Reply:
To be clear, this is managed via a bot... Which I'm about to use :)